web_src/less/_base.less

```diff
@@ -1122,11 +1137,6 @@ i.icon.centerlock {
   table {
     width: 100%;
   }
-
-  .active {
```

Comment:
this would break blame line highlight since it is structured different -- maybe need a separate rule above to handle that too

Reply:
fixed, thanks